cloudify_marketplace/resources/cloudify_security_reconfiguration

```diff
@@ -0,0 +1,267 @@
+#! /usr/bin/env python
+import json
+import logging
+import logging.handlers
+import os
+import re
+import shutil
+import subprocess
+import time
+import yaml
+
+from cloudify_rest_client import CloudifyClient
+from cloudify_cli.utils import get_auth_header
+
+target_file = '/tmp/cloudify_ssl_certificate_replacement.json'
+
+# Config paths
+MGMTWORKER_CONFIG_PATH = '/opt/mgmtworker/work/broker_config.json'
+RESTSERVICE_CONFIG_PATH = '/opt/manager/cloudify-rest.conf'
+AMQPINFLUX_CONFIG_PATH = '/etc/sysconfig/cloudify-amqpinflux'
+MANAGER_REST_SECURITY_FILE = '/opt/manager/rest-security.conf'
+LOGSTASH_CONFIG_PATH = '/etc/logstash/conf.d/logstash.conf'
+RIEMANN_CONFIG_PATH = '/etc/sysconfig/cloudify-riemann'
+
+# Executable paths
+RABBITMQCTL = '/sbin/rabbitmqctl'
+SED = '/bin/sed'
+
+# Broker service
+BROKER_SERVICE_NAME = 'cloudify-rabbitmq'
+
+# Other Services
+RESTART_SERVICES = [
+    'nginx',
+    # web server
+    'cloudify-amqpinflux',
+    # mgmtworker
+    'cloudify-mgmtworker',
+    # manager
+    'cloudify-restservice',
+    'cloudify-riemann',
+    # metrics handler
+    'logstash',
+    # log/event handler
+    ]
+
+logger = logging.getLogger('cloudify_security_config')
+# TODO: logging.INFO or maybe look for a flag
+logger.setLevel(logging.DEBUG)
+handler = logging.handlers.SysLogHandler(address='/dev/log')
+logger.addHandler(handler)
+
+
+def service(action, name):
+    """Run `action` on the service `name`"""
+    logger.info('{action}ing {service}'.format(action=action, service=name))
+    # If it fails to stop that's not a critical problem
```

Comment:
We haven't... and I'm not sure what we would do even if they did fail, so maybe just comment to that effect, e.g. a better worded version of:
# We don't mind if stopping fails as it will lead to logged warnings and errors but not actually cause real problems
# When restarting the services we're not in a position to report back the failure in any sensible way, so we will continue